samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -132,112 +137,167 @@ public class Assignment
+                }
+                catch (Exception ex)
+                {
+                    throw new GraphException($"SetupResourcesFolder call: {ex.Message}", ex, classId, assignmentId);
+                }
             }
-            catch (Exception ex)
+
+            /// <summary>
+            /// Deactivate an assignment, changes the state of an educationAssignment from its original draft status to the Inactive status
+            /// Reference :: https://learn.microsoft.com/en-us/graph/assignments-states-transition
+            /// </summary>
+            /// <param name="client">Microsoft Graph service client</param>
+            /// <param name="classId">User class id</param>
+            /// <param name="assignmentId">Assignment id in the class</param>
+            /// <returns>EducationAssignment</returns>
+            public static async Task<EducationAssignment> DeactivateAsync(
+                GraphServiceClient client,
+                string classId,
+                string assignmentId)
             {
-                throw new GraphException($"PostResourceAsync call: {ex.Message}", ex, classId, assignmentId);
+                try
+                {
+                    return await client.Education
+                        .Classes[classId]
+                        .Assignments[assignmentId]
+                        .Deactivate.PostAsync(requestConfig =>
+                        {
+                            requestConfig.Headers.Add(
+                                "Prefer", "include-unknown-enum-members");
+                        });
+                }
+                catch (Exception ex)
+                {
+                    throw new GraphException($"DeactivateAsync call: {ex.Message}", ex, classId, assignmentId);
+                }
             }
-        }
 
         /// <summary>
-        /// Sets up the assignment resources folder
+        /// Create Assignment Feedback resources folder
+        /// Reference :: https://learn.microsoft.com/en-us/graph/assignments-states-transition
         /// </summary>
         /// <param name="client">Microsoft Graph service client</param>
         /// <param name="classId">User class id</param>
-        /// <param name="assignmentId">Assignment id</param>
+        /// <param name="assignmentId">Assignment id in the class</param>
         /// <returns>EducationAssignment</returns>
-        public static async Task<EducationAssignment> SetupResourcesFolder(
-            GraphServiceClient client,
-            string classId,
-            string assignmentId)
-        {
-            try
-            {
-                return await client.Education
-                    .Classes[classId]
-                    .Assignments[assignmentId]
-                    .SetUpResourcesFolder
-                    .PostAsync();
-            }
-            catch (Exception ex)
+        public static async Task<EducationAssignment> SetUpAssignmentFeedbackResourcesFolder(
```

Comment:
SetUpAssignmentFeedbackResourcesFolderAsync

Reply:
updated "SetUpAssignmentFeedbackResourcesFolder" to "SetUpAssignmentFeedbackResourcesFolderAsync"